README.md

```diff
@@ -66,6 +66,16 @@ pip_import(
 
    # set compile to true only if requirements files is not already compiled
    # compile = True
+
+   # add extra build files to be added to extend the `rules_pip`.
+   # the content of the BUILD file is extended by appending the
+   # the content from the supplied file to the generated file for a pip
+   # pacakage.
+   # extra_build_files = {"tensorflow": "@//:src/tensorflow.BUILD"},
+
+   # add aliases for the extra targets defined in the build file above.
```

Comment:
this is not needed IMO, just append the BUILD file to generated build file and keep things simple. the target is accessible using `requirements`, right?

Reply:
`requirement` does work yes. But this avoids having to add a redundant `load` statement in BUILD files across the repository. I suggest we keep it.

Reply:
we can always add that with a better design, current implementation will pollute @pip_reqs repos, we need some hierarchical design for it, instead of adding all the targets to the root. So let's only keep adding the BUILD file for now